doc/protocol.md

```diff
@@ -289,11 +289,16 @@ is no longer supported.
 Additional "open" command options are needed to open a channel of this
 type:
 
- * "bus": The DBus bus to connect to either "session" or "system",
-   defaults to "system" if not present. If set to "internal" then this
-   channel will communicate with the internal bridge DBus connection.
+ * "bus": The DBus bus to connect to either "session", "system", "none"
+   or "internal". Defaults to "system" if not present. If set to "none"
+   you must also provide the address parameter. If set to "internal"
```

Comment:
Lets not document "internal" here. We do need to support it, but it's a deprecated thing.